### ISSUE TITLE
Include timestamp, and ip address in exeception report

### ISSUE THREAD

**Issue by jsieber** [gregmoser/MuraHoth#5]
I'd like to add a time stamp and ip address to the exception reports, but not sure if it is possible based on how the exceptions are grouped. Anyone else think this would be a good addition?

**Comment by gregmoser** [comment]
Sounds like a good idea to me.  My understanding is that this might be on the individual exceptions but not the group of exceptions.

**Comment by m-evangelista** [comment]
Yes (and if not already included, I'd suggest we grab the user-agent too). When you start getting errors from a single bot or hacking attempt, having the IP and agent type is super useful.

@jsieber-aca are you working on this, or just suggesting it?

**Comment by jsieber** [comment]
@gowestweb - I'd like to work on it, but have no idea how soon I will be able to take a stab at it. If I get started on it I'll update here. Also agree about the user agent information.
